framework/src/play/deps/DependenciesManager.java

```diff
@@ -26,18 +26,38 @@
 import play.libs.IO;
 
 public class DependenciesManager {
-   
+
     public static void main(String[] args) throws Exception {
+        String applicationPath = System.getProperty("application.path");
```

Comment:
It will avoid exception if we used incorrectly the method, this way we know that all this variable are required